applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -0,0 +1,372 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "bt_mgmt.h"
+
+#include <zephyr/zbus/zbus.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/hci.h>
+#include <zephyr/settings/settings.h>
+#include <zephyr/sys/byteorder.h>
+
+#include "macros_common.h"
+#include "nrf5340_audio_common.h"
+#include "ble_audio_services.h"
+#include "button_handler.h"
```

Comment:
Remove button functionality from bt_mgmt files

Reply:
After looking at it it is a bit more involved than originally thought, so will rather make a task for it to do later